documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -4,6 +4,31 @@
 Release Notes
 #############
 
+7.1.9
+=====
+* Same as 7.1.8 release with AVX enabled.
+
+7.1.8
+=====
+* Released with AVX disabled.
+* Fixed a performance regression in network run loop.  `(PR #7342) <https://github.com/apple/foundationdb/pull/7342>`_
+* Added RSS bytes for processes in status json output and corrected available_bytes calculation. `(PR #7348) <https://github.com/apple/foundationdb/pull/7348>`_
+* Added versionstamp support in tuples. `(PR #7313) <https://github.com/apple/foundationdb/pull/7313>`_
+* Fixed some spammy trace events. `(PR #7300) <https://github.com/apple/foundationdb/pull/7300>`_
+* Fixed a memory corruption bug for using streaming peeks. `(PR #7288) <https://github.com/apple/foundationdb/pull/7288>`_
```

Comment:
This isn't quite right, this PR does not fix the memory corruption bug, it just avoids it by disabling streaming peek.

Reply:
I reworded this in #7468.